about_first_project.css

```diff
@@ -0,0 +1,46 @@
+body {
+    background-color: #C9F4EB;
+    margin: 0px; 
+}
+
+h1 {
+    font-size: 70px;
+    text-align: center;
+    color: black;
+    font-family: 'Nunito', sans-serif;
+    font-style: oblique;
+    padding-top: 40px;
+    }
```

Comment:
A general comment on the indentation in CSS is to move `}` to the left (for example as on line 4).